SmartDeviceLink/SDLIAPSession.m

```diff
@@ -141,10 +258,26 @@ - (SDLStreamErrorHandler)streamErroredHandler {
     };
 }
 
+- (SDLStreamHasSpaceHandler)streamHasSpaceHandler {
```

Comment:
Joel, I am happy to make the private methods I've added conform to the naming convention. I think it's out of the scope of this PR to rename them all. Do you agree?